html/mix-manifest.json

```diff
@@ -1,12 +1,8 @@
 {
-    "/js/app.js": "/js/app.js?id=6a5160d48cf9260d77e1",
-    "/css/app.css": "/css/app.css?id=2a88c6515df894dc6f36",
-    "/js/manifest.js": "/js/manifest.js?id=3c768977c2574a34506e",
-    "/js/vendor.js": "/js/vendor.js?id=58c8fc0774b5843ec004",
-    "/js/lang/de.js": "/js/lang/de.js?id=8c1c390a5f45ac01d54b",
-    "/js/lang/en.js": "/js/lang/en.js?id=201d470a19d7fa80027b",
-    "/js/lang/fr.js": "/js/lang/fr.js?id=807a43a37f4eeb4c1d17",
-    "/js/lang/ru.js": "/js/lang/ru.js?id=1a4435677a27aa615af1",
-    "/js/lang/uk.js": "/js/lang/uk.js?id=6ce358a9e9387947ee03",
-    "/js/lang/zh-TW.js": "/js/lang/zh-TW.js?id=3ad4a3e6a5a59165e806"
+    "/js/lang/de.js": "/js/lang/de.js?id=b73bd81531880879e765",
```

Comment:
You removed the other files from the manifest somehow.

Reply:
I really don't get how this works. I only left the php files and nothing else now. What else should I do ?

Reply:
Run `npm run prod` should rebuild the entire file.

Reply:
Also, it doesn't matter what the manifest key is as long as it is different when the file changes otherwise, the browser will keep using a cached version of the file.

Reply:
npm seems to be unhappy. That's why the resulting file is not correct: 
```
librenms@librenms:~$ cat   /opt/librenms/.npm/_logs/2020-03-17T08_41_19_366Z-debug.log
0 info it worked if it ends with ok
1 warn npm npm does not support Node.js v10.15.2
2 warn npm You should probably upgrade to a newer version of node as we
3 warn npm can't make any promises that npm will work with this version.
4 warn npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
5 warn npm You can find the latest version at https://nodejs.org/
6 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'production' ]
7 info using npm@5.8.0
8 info using node@v10.15.2
9 verbose config Skipping project config: /opt/librenms/.npmrc. (matches userconfig)
10 verbose run-script [ 'preproduction', 'production', 'postproduction' ]
11 info lifecycle @~preproduction: @
12 info lifecycle @~production: @
13 verbose lifecycle @~production: unsafe-perm in lifecycle true
14 verbose lifecycle @~production: PATH: /usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/opt/librenms/node_modules/.bin:/usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/opt/librenms/node_modules/.bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games
15 verbose lifecycle @~production: CWD: /opt/librenms
16 silly lifecycle @~production: Args: [ '-c',
16 silly lifecycle   'cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js' ]
17 silly lifecycle @~production: Returned: code: 2  signal: null
18 info lifecycle @~production: Failed to exec production script
19 verbose stack Error: @ production: `cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
19 verbose stack Exit status 2
19 verbose stack     at EventEmitter.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/index.js:283:16)
19 verbose stack     at EventEmitter.emit (events.js:189:13)
19 verbose stack     at ChildProcess.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
19 verbose stack     at ChildProcess.emit (events.js:189:13)
19 verbose stack     at maybeClose (internal/child_process.js:970:16)
19 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:259:5)
20 verbose pkgid @
21 verbose cwd /opt/librenms
22 verbose Linux 5.3.0-40-generic
23 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "production"
24 verbose node v10.15.2
25 verbose npm  v5.8.0
26 error code ELIFECYCLE
27 error errno 2
28 error @ production: `cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js`
28 error Exit status 2
29 error Failed at the @ production script.
29 error This is probably not a problem with npm. There is likely additional logging output above.
30 verbose exit [ 2, true ]
```